ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -567,3 +567,36 @@ def get_blockpools_compression_status_from_storagesystem(self) -> tuple:
             self.validation_loc["storagecluster-blockpool-details-compress-status"]
         )
         return compression_status_blockpools_tab, compression_status_blockpools_details
+
+    def verify_odf_operator_in_installed_operator(self) -> bool:
+        """
+        Function to validate either ODF operator pod is up and running post ODF installation,
```

Comment:
I did not find any validation of odf operator pod post odf installation

Reply:
This validation is not regarding pod, it is just to check whether the ODF operator is only visible in installed operator tab in ui

Reply:
The doc text still says `ODF operator pod is up and running` but there is no validation for `pod` as you said. So remove the word `pod` from the doc string.